### PR TITLE
FB-339 New purchase process

### PR DESCRIPTION
**Pull request by piotrpospiech**
- created trial page with license agreement
- added license agreement for paid plans (it is used in Stripe Payment Link)
- added Stripe Payment Links
- added email template
- fixed tsc config

### CODE REVIEW

File: emails/source-code-download.html [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+              </tr>
+            </tbody>
+          </table>
+        </td>
+      </tr>
+    </tbody>
+  </table><!-- End -->
+</body>
+
+</html>
```
Comment (mkaliszewski) [inline emails/source-code-download.html:238]: No empty line EOF

**Review by mkaliszewski — CHANGES_REQUESTED** [review]
1. Is there any chance we can configure the stripe page to add a `Back home` button or sth similar after purchase? I never used it, so I'm just asking.
Screenshot from 2023-09-22 14-37-44

2. I selected trial and got an email from `piotr.pospiech@vazco.eu` email address. It may not be correct.

3. Images are not present in the email
Screenshot from 2023-09-22 14-37-10

File: emails/source-code-download.html (hunk 1 above)
Comment (piotrpospiech) [reply emails/source-code-download.html:238]: It is default formatting for html. I will leave it as it is.

**Comment by piotrpospiech** [comment]
@mkaliszewski 

1. We can display message after completing checkout or redirect user. We decided to display a message.
2. We will configure forminer email later. We want to deploy it with this email for now.
3. Images will be displayed after deploy as they are static assets in the forminer website. The same is for the paid plan license which link is available in the Stripe checkout.

**Review by mkaliszewski — APPROVED** [review]

**Pull request merged**